### PR TITLE
OCPBUGS-43323:adds warning message for IPsec updates to 4.15

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-43323

Link to docs preview:
https://83462--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/network_security/configuring-ipsec-ovn.html

QE review:
- [x] QE has approved this change.

Additional information:
PTAL: @zshi-redhat

### CODE REVIEW

File: networking/network_security/configuring-ipsec-ovn.adoc [hunk 1]
```diff
@@ -13,6 +13,8 @@ IPsec is disabled by default. It can be enabled either during or after installin
 [IMPORTANT]
 ====
 If your cluster uses link:https://www.redhat.com/en/topics/containers/what-are-hosted-control-planes[{hcp}] for Red Hat {product-title}, IPsec is not supported for IPsec encryption of either pod-to-pod or traffic to external hosts.
+
+You must disable IPsec before updating to {product-title} 4.15. There is a known issue that can cause interruptions in pod-to-pod communication if you update without disabling IPsec.
```
Comment (zshi-redhat) [inline networking/network_security/configuring-ipsec-ovn.adoc:17]: please add link to the known issue: https://issues.redhat.com/browse/OCPBUGS-41823

Comment (zshi-redhat) [reply networking/network_security/configuring-ipsec-ovn.adoc:17]: @pperiyasamy  PTAL if it's accurate to say pod-to-pod communication will be interrupted.

Comment (JoeAldinger) [reply networking/network_security/configuring-ipsec-ovn.adoc:17]: I can't link to a bug that is RH security. Please provide CCFR in this bug: https://issues.redhat.com/browse/OCPBUGS-43323. I will rewrite it and then link to it.

Comment (zshi-redhat) [reply networking/network_security/configuring-ipsec-ovn.adoc:17]: what is CCFR?

Comment (JoeAldinger) [reply networking/network_security/configuring-ipsec-ovn.adoc:17]: cause, consequence, fix, and result

Comment (pperiyasamy) [reply networking/network_security/configuring-ipsec-ovn.adoc:17]: the content looks ok to me.

**Comment by JoeAldinger** [comment]
> I think this warning should be also documented in the "Preparing to update to OpenShift Container Platform 4.15" doc [1], and in the "Known Issues" section of the "Release Notes" [2].
> 
> [1] [docs.openshift.com/container-platform/4.15/updating/preparing_for_updates/updating-cluster-prepare.html](https://docs.openshift.com/container-platform/4.15/updating/preparing_for_updates/updating-cluster-prepare.html) [2] [docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-known-issues](https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-known-issues)

Sorry, I should've linked the PRs in the main comment: 
https://github.com/openshift/openshift-docs/pull/83462#issuecomment-2419584459

**Comment by oarribas** [comment]
@JoeAldinger , the current PR is only for `networking/network_security/configuring-ipsec-ovn`. My comment is about it should be also in the other pages, the "Preparing to update to OpenShift Container Platform 4.15" and the 4.15 "Release Notes".

**Comment by JoeAldinger** [comment]
> @JoeAldinger , the current PR is only adding a note in the `networking/network_security/configuring-ipsec-ovn` document. My comment is to add the note (and maybe additional information) to the other pages, the "Preparing to update to OpenShift Container Platform 4.15" [1] and the 4.15 "Release Notes" [2], as they are the best places for warn about upgrade issues.
> 
> [1] [docs.openshift.com/container-platform/4.15/updating/preparing_for_updates/updating-cluster-prepare.html](https://docs.openshift.com/container-platform/4.15/updating/preparing_for_updates/updating-cluster-prepare.html) [2] [docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-known-issues](https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-known-issues)

@obrown1205 wdyt about putting this in the updating docs? I've never seen a KI placed there so I'm looking for some guidance.

**Comment by JoeAldinger** [comment]
> @JoeAldinger , the current PR is only adding a note in the `networking/network_security/configuring-ipsec-ovn` document. My comment is to add the note (and maybe additional information) to the other pages, the "Preparing to update to OpenShift Container Platform 4.15" [1] and the 4.15 "Release Notes" [2], as they are the best places for warn about upgrade issues.
> 
> [1] [docs.openshift.com/container-platform/4.15/updating/preparing_for_updates/updating-cluster-prepare.html](https://docs.openshift.com/container-platform/4.15/updating/preparing_for_updates/updating-cluster-prepare.html) [2] [docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-known-issues](https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-known-issues)

My other PR that I provided places this in the RNs. I'm checking with a colleague about the updating docs.

**Comment by obrown1205** [comment]
@JoeAldinger I think it makes sense to have a warning in the Preparing to update section if it's an issue we are aware of. I've added some warnings before. If I understand the known issue correctly, the issue only occurs when updating from 4.14 --> 4.15? Not 4.15+?

**Comment by JoeAldinger** [comment]
> @JoeAldinger I think it makes sense to have a warning in the Preparing to update section if it's an issue we are aware of. I've added some warnings before. If I understand the known issue correctly, the issue only occurs when updating from 4.14 --> 4.15? Not 4.15+? Do we know the cause of the bug or specifics? Is that something we'd want to publish to customers aside from calling it "an issue"?

cc @zshi-redhat and @anuragthehatter do we have an update on the CCFR?

**Comment by zshi-redhat** [comment]
> > @JoeAldinger I think it makes sense to have a warning in the Preparing to update section if it's an issue we are aware of. I've added some warnings before. If I understand the known issue correctly, the issue only occurs when updating from 4.14 --> 4.15? Not 4.15+? Do we know the cause of the bug or specifics? Is that something we'd want to publish to customers aside from calling it "an issue"?
> 
> cc @zshi-redhat and @anuragthehatter do we have an update on the CCFR?

@pperiyasamy hi, could you please help to add CCFR (cause, consequence, fix, and result)? we need to get this doc update out asap.

**Comment by pperiyasamy** [comment]
> @pperiyasamy hi, could you please help to add CCFR (cause, consequence, fix, and result)? we need to get this doc update out asap.

talked with @JoeAldinger offline and provided info for CCFR.

**Comment by ahardin-rh** [comment]
LGTM. Thank you!

**Review by dfitzmau — APPROVED** [review]
The release note doc update is covered under https://github.com/openshift/openshift-docs/pull/83463/files

**Pull request merged**